{
    "identifier": "CVE-2019-10682",
    "summary": "Django-nopassword before 5.0.0 stores cleartext secrets in the database. See: CVE-2019-10682.",
    "references": [
        {
            "url": "",
            "reference_id": "pyup.io-38080"
        }
    ],
    "vulnerable_packages": [
        "pkg:pypi/django-nopassword@2.0.0",
        "pkg:pypi/django-nopassword@3.0.0",
        "pkg:pypi/django-nopassword@2.1.1",
        "pkg:pypi/django-nopassword@4.0.0",
        "pkg:pypi/django-nopassword@3.0.2",
        "pkg:pypi/django-nopassword@0.4",
        "pkg:pypi/django-nopassword@0.7.0",
        "pkg:pypi/django-nopassword@0.3.3",
        "pkg:pypi/django-nopassword@2.1.0",
        "pkg:pypi/django-nopassword@3.0.1",
        "pkg:pypi/django-nopassword@1.2.0",
        "pkg:pypi/django-nopassword@0.4.1",
        "pkg:pypi/django-nopassword@0.4.3",
        "pkg:pypi/django-nopassword@1.3.0",
        "pkg:pypi/django-nopassword@0.6.0",
        "pkg:pypi/django-nopassword@0.2",
        "pkg:pypi/django-nopassword@0.3.1",
        "pkg:pypi/django-nopassword@0.1.2",
        "pkg:pypi/django-nopassword@0.4.2",
        "pkg:pypi/django-nopassword@0.5.0",
        "pkg:pypi/django-nopassword@0.3.2",
        "pkg:pypi/django-nopassword@4.0.2",
        "pkg:pypi/django-nopassword@1.3.1",
        "pkg:pypi/django-nopassword@0.3",
        "pkg:pypi/django-nopassword@4.0.1",
        "pkg:pypi/django-nopassword@0.8.0",
        "pkg:pypi/django-nopassword@1.0.0",
        "pkg:pypi/django-nopassword@1.1.0"
    ],
    "resolved_packages": [
        "pkg:pypi/django-nopassword@5.0.0"
    ]
}